test/e2e/data/infrastructure-docker/cluster-template-mhc.yaml

```diff
@@ -1,147 +1,154 @@
-apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
-kind: DockerCluster
-metadata:
-  name: '${ CLUSTER_NAME }'
----
-apiVersion: cluster.x-k8s.io/v1alpha3
-kind: Cluster
-metadata:
-  name: '${ CLUSTER_NAME }'
-  labels:
-    cni: "${CLUSTER_NAME}-crs-0"
-spec:
-  clusterNetwork:
-    services:
-      cidrBlocks: ['${ DOCKER_SERVICE_CIDRS }']
-    pods:
-      cidrBlocks: ['${ DOCKER_POD_CIDRS }']
-    serviceDomain: '${ DOCKER_SERVICE_DOMAIN }'
-  infrastructureRef:
-    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
-    kind: DockerCluster
-    name: '${ CLUSTER_NAME }'
-  controlPlaneRef:
-    kind: KubeadmControlPlane
-    apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
-    name: "${CLUSTER_NAME}-control-plane"
----
-apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
-kind: DockerMachineTemplate
-metadata:
-  name: "${CLUSTER_NAME}-control-plane"
-spec:
-  template:
-    spec:
-      extraMounts:
-        - containerPath: "/var/run/docker.sock"
-          hostPath: "/var/run/docker.sock"
----
-kind: KubeadmControlPlane
-apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
+apiVersion: v1
```

Comment:
I would like to gitignore the generated cluster-templates and keep under source control only the bases, but it seems that this makes the apidiff batch to fail 🤔